react/Modal.tsx

```diff
@@ -91,9 +98,12 @@ const Modal: React.FC<Props> = props => {
     <BaseModal
       open={open}
       backdrop={backdrop}
+      onClose={handleClose}
       onBackdropClick={handleBackdropClick}
     >
-      <div className={containerClasses}>
+      {/* click-events-have-key-events is disabled because this will be handled by BaseModal */}
+      {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events */}
```

Comment:
⛏[Nitpick] I would move this to the top of the file. I know it disables the rule for the whole file, but that's much cleaner and this is a very specific rule